release-engineering/role-handbooks/branch-manager.md

```diff
@@ -235,129 +225,105 @@ After having thoroughly read the section on cutting a release version of the han
 
 #### Alpha Stage
 
-##### gcbmgr stage
+##### krel gcbmgr stage
 
 Start staging a build by running:
 
 ```shell
-./gcbmgr stage master
+krel gcbmgr --stage --branch master --build-version=$(curl -Ls https://dl.k8s.io/ci/latest.txt)
 ```
-
 This command should return relatively quickly and provide a link to GCP where you can track the process of the build.
+This takes time (approximately 1 hour is the current norm). It’s building all the bits for a bunch of target operating systems and hardware architectures.
+
+<!-- TODO: krel gcbmgr is not checking testgrid
 
 Early in the release cycle, it is likely that the build might fail. By default the `gcbmgr stage master` command automatically looks for a place where [release master blocking tests](https://k8s-testgrid.appspot.com/sig-release-master-blocking) have green results, which traditionally has not happened in Kubernetes on an ongoing basis.
 
 WE REALLY WANT (and need) TO GET THERE. Quality needs to be a continual focus. But in the meantime, acknowledging today especially for an early alpha or beta release, it is possible to just build via:
 
-```shell
-./gcbmgr stage master
-```
-
-This will give you a suggested `--buildversion`.  Use this as:
-
-```shell
-./gcbmgr stage master --buildversion=v1.16.0-alpha.0.1+704790e0412842
-```
-
-or can use using the file produced but the testing automation:
-
-```shell
-./gcbmgr stage master --buildversion=$(curl -Ls https://dl.k8s.io/ci/latest.txt)
-```
-
 Rather than having `gcbmgr` pick a candidate by analyzing test data from the commit history that had no fails and building automatically from that point, we instead indicate we want to build explicitly from the last commit on the current branch.
+-->
 
-This takes time (approximately 1 hour is the current norm). It’s building all the bits for a bunch of target operating systems and hardware architectures.
-
-##### gcbmgr list
+##### krel gcbmgr list
 
 ```shell
-./gcbmgr list
+krel gcbmgr --branch master
```

Comment:
Should we default to "master" in krel?

Reply:
no strong opinions, but agree

Reply:
See https://github.com/kubernetes/release/pull/1205